File: Codigos em C/Lista 07-26/3. De 5 somar media dos impares.c
#include <stdio.h>

int main(){

int i=1,m=0, a;
float n;
while( i <= 5  ){
printf ("Informe um numero: \n");
scanf ("%i", &a);
if (a % 2 != 0){
    n = n + a;
    m++;
    
}
i=i+1;
}
n = n/m;
printf("a a media dos impares eh: %.1f \n",n );
return 0;
}

File: Conversão para Python/Lista 07-26/3. De 5 somar media dos impares.py

i=1
n=0
m=0
while i <= 5 :
    a = int(input("Informe um numero: "))
    if (a % 2 != 0):
        n = n + a
        m = m +1
    
    i=i+1
print("a media dos impares eh: ", n/m)

